### ISSUE TITLE
Create Landscape files using Command Line

### ISSUE THREAD

**Issue by aklibenali** [firelab/farsite#6]
Is there any function to create landscape files (lcp) in an "automated way"? 

In a previous FARSITE CLI version there was a "lcpmake", which was quite handy.

**Comment by latwood** [comment]
I strongly doubt it. I don't remember seeing anything like this
unfortunately. In fact, I think one of the major differences/problems that
needs fixed between the different versions of farsite has to do with
projection info related to lcp stuff, I think this version ignores it but
the windows version does some kind of in between. The most updated version,
which is in flammap, does use gdal and other complex stuff to do the lcp
stuff and projections quite well, but I haven't played with it yet. When I
did the work on this version of farsite, the windows version was still
available and that was what I was using to build all my lcp files. I would
build them directly in the windows farsite gui, same with the ignition
files, then use them in this cli version.

**Comment by aklibenali** [comment]
I want to simulated around 60 wildfires, so I have to create 60 landscape files (actually it is much more, but nevermind). Given the amount of lcps, it's not feasible to do them using FARSITE gui.

That Flammap version, is the routine to create the lcps available? Is it for Linux?

**Comment by latwood** [comment]
Yeah you definitely will want a cli style tool for that many fires if you
can. Unfortunately the flammap I'm talking about is for windows. Here's the
link to where the windows farsite used to be:
https://www.firelab.org/project/farsite. You'll notice that when you go to
software, it isn't available anymore. Here's the link to flammap:
https://www.firelab.org/project/flammap. Not sure if that version has as
nice a gui for some of the farsite stuff as the old farsite, but it is the
only available farsite related gui that I know of.

And I made a mistake. I think farsite had lcp file making tools, but I
always made my lcp files in WindNinja, and the ignition files in farsite.
Can try it out using the WindNinja gui first to get a rough idea how
building an lcp file works, then should be able to use the WindNinja cli.
Might even be able to get by just by reading some of the user guides for
WindNinja as well. If I remember correctly, WindNinja pretty much just
downloads the file from the landfire database with a hint of processing. If
you need more capability then that, you'll probably need to write your own
tool, or ask around to find if anyone has such a tool they've already
written.

**Comment by latwood** [comment]
I also did a small project trying to write my own code to automate some of
the windninja to farsite stuff, found here:
https://github.com/latwood/WRF-WindNinja-FarsiteScript. But I never quite
got to the point where I did the lcp downloading and ignition file creation
done inside this code, I was still manually creating those files from
WindNinja and the windows version of farsite. That was the next part of the
project, which had to put off.
